### PR TITLE
e2e/setup cypress

### PR DESCRIPTION
**Pull request by francisprovost**
- Install and setup Cypress
- Basic documentation for the commands
- Basic first tests that make sure the patient search screens loads after login

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import './commands';
+
+// Ignore uncaught exception so tests doesn't stop mid run
+Cypress.on('uncaught:exception', () => false);
```
Comment (YanLabCHU) [inline cypress/support/index.js:4]: Can we log the exception first? we could still ignore it afterwards

File: cypress/support/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (YanLabCHU) [inline cypress/support/index.d.ts:1]: ?

**Review by YanLabCHU — COMMENTED** [review]
Juste quelques commentaires

File: cypress/support/index.js (hunk 1 above)
Comment (francisprovost) [reply cypress/support/index.js:4]: It still logs it in the browser console. It's just that, by default, as soon as there's an uncaught exception the tests fails and sadly, there's currently one just before keycloak redirects us to the app after the login. I would prefer to fix the error but it's not the best timing and "it works" so..

File: cypress/support/index.d.ts (hunk 2 above)
Comment (francisprovost) [reply cypress/support/index.d.ts:1]: There's something weird where `eslint` says `Chainable` is not used (even though it is). Even in the lib `cypress-testing-library` they added it for the same reason.

**Review by YanLabCHU — APPROVED** [review]
Good job!

**Pull request merged**